### PR TITLE
Update of environments article

### PR DESCRIPTION
**Pull request by sweetsweetstefan**
#### Objective

Review the updated article for accuracy and completeness.  Right now there are no screenshots but these will be added later.

### CODE REVIEW

File: en/basic-concepts/environments-to-organize-workloads-and-users.md [hunk 1]
```diff
@@ -3,12 +3,48 @@ title: "Using Environments to organize users & workloads"
...
+   - Configure no network (click *Skip* to finish creating the environment)
+1. If you've chosen to create one or more networks, enter the requested parameters for the networks to create.
+1. Click *Initialize*.
+1. The networks will be created and you will be returned to the *Environments* page.
+
+### Manage and delete an environment
+
+From the *Environments* page, you can click on the *Edit* icon to the right of an environment to change the name, the description, or the option to allow external members.
+
+Clicking on the *Action* menu for an environment will allow you to manage members who have access to the environment, or to delete the environment.  Additionally, you can copy the environment's UUID from the underlying cloud orchestrator, and you can also force cached information on this environment to be refreshed from the cloud orchestrator.
```
Comment (Shabirmean) [inline en/basic-concepts/environments-to-organize-workloads-and-users.md:48]: **_Suggestion for the last line_**
- Additionally, you can also copy the environment's UUID from the underlying cloud orchestrator, and force cached information on this environment to be refreshed from the cloud orchestrator.

File: fr/administration/rbac.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Pour contrôler l'accès aux ressources dans un environnement, CloudMC conçoit
 
 ## Comment assigner le rôles
 
-CloudMC permet à un utilisateur avec le rôle *Administrateur* et plus élevé (ou un utilisateur avec un rôle personnalisé incluyant la permission *Utilisateurs : Gérer*) à assigner des rôles fixes et personnalisés aux utilisateurs.  L'administrateur n'est pas permis à accorder une permission à un autre utilisateur sans déjà avoir cette même permission.  Ce modèle de sécurité empêche un utilisateur d'augmenter son propre accès sans authorisation (élévation des privilèges).
+CloudMC permet à un utilisateur avec le rôle *Administrateur* et plus élevé (ou un utilisateur avec un rôle personnalisé incluyant la permission *Utilisateurs : Gérer*) à assigner des rôles fixes et personnalisés aux utilisateurs.  L'administrateur n'est pas permis à accorder une permission à un autre utilisateur sans déjà avoir cette même permission.  Ce modèle de sécurité empêche un utilisateur d'augmenter son propre accès sans autorisation (élévation des privilèges).
```
Comment (Shabirmean) [inline fr/administration/rbac.md:80]: This looks good to be 😅

File: en/basic-concepts/environments-to-organize-workloads-and-users.md [hunk 3]
```diff
@@ -3,12 +3,48 @@ title: "Using Environments to organize users & workloads"
...
+### Environments overview
 
-Although CloudMC calculates service usage at the organization-level for billing purpose, the resources consumed by each environments are also tracked separately, which allows businesses to generate internal chargeback reports on a per-environment basis if they wish.
+CloudMC provides a powerful mechanism, **environments**, to segregate workloads/ressources and to control who gets access to those. A typical use case of this concept is to create distinct environments to isolate production workloads from development systems, or to establish project-specific [sandboxes](https://en.wikipedia.org/wiki/Sandbox_%28computer_security%29).
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply en/basic-concepts/environments-to-organize-workloads-and-users.md:9]: Thank you for catching these, they came over from the original, I changed the sentence a bit, you're right it wasn't great.

File: en/basic-concepts/environments-to-organize-workloads-and-users.md [hunk 4]
```diff
@@ -3,12 +3,48 @@ title: "Using Environments to organize users & workloads"
...
 
-## Environment Roles
-Furthermore, the environment membership is also coupled with an environment role, which controls what a user can do with the resources contained in this environment. Contrary to [System](system-roles.md) roles, the Environment roles are fixed and cannot (yet) be customized granularly.
+An environment belongs to an organization, is associated with a specific service (e.g.: Compute or Object Storage), and is comprised of a set of users who have visibility on a pool of shared resources (e.g.: instances, storage, etc). New CloudMC organizations are provisioned with default environments, which are accessible to all their users.
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply en/basic-concepts/environments-to-organize-workloads-and-users.md:11]: Yup, took it out and improved the prior sentence as well.

File: en/basic-concepts/environments-to-organize-workloads-and-users.md [hunk 5]
```diff
@@ -3,12 +3,48 @@ title: "Using Environments to organize users & workloads"
...
+
+Although CloudMC calculates service usage at the organization-level for billing purposes, the resources consumed by each environments are also tracked separately, which allows businesses to generate internal chargeback reports on a per-environment basis if they wish.
+
+Navigate to the desired service in the sidebar to get to the *Environments* page, where all the environments in a service are listed.  Additionally, when working inside of an environment the current service and environment are displayed at the top of the page on the left side.  Clicking on this button will display an option to return to the *Environments* page, or to switch to another environment.
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply en/basic-concepts/environments-to-organize-workloads-and-users.md:15]: Awesome improvements, applied.  This paragraph will also benefit from a screenshot eventually.

File: en/basic-concepts/environments-to-organize-workloads-and-users.md [hunk 6]
```diff
@@ -3,12 +3,48 @@ title: "Using Environments to organize users & workloads"
...
+
+### Environment Roles
+
+Environment membership is coupled with an **environment role**, which controls what a user can do with the resources contained in this environment. Contrary to system roles, the environment roles are fixed and cannot be customized granularly.
+
+### Create an environment
+
+Any user with the **User** role or the *Environments: Create* permission can add additional environments and decide who should have access to it.
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply en/basic-concepts/environments-to-organize-workloads-and-users.md:23]: I like it! Done.

File: en/basic-concepts/environments-to-organize-workloads-and-users.md [hunk 7]
```diff
@@ -3,12 +3,48 @@ title: "Using Environments to organize users & workloads"
...
+1. Click on *Add environment*.  The *Add environment* page will appear.
+1. Enter a name and an optional description for the environment.
+1. If you wish to allow users from other organizations to be added as members, select **Allow external members**.
+1. Click *Submit*.  The environment will be provisioned.
+1. The *Manage members* page appears.  Here you may:
+   - select one or more individual users from the list
+   - search for specific users
+   - add **all users** in the organization (auto-membership)
+   - add no users to your environment (click *Skip* to proceed to the next page)
+1. For any users you've added, select an environment role, and if you've enabled auto-membership, select a default environment role.
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply en/basic-concepts/environments-to-organize-workloads-and-users.md:34]: Agreed! Done.

File: en/basic-concepts/environments-to-organize-workloads-and-users.md [hunk 8]
```diff
@@ -3,12 +3,48 @@ title: "Using Environments to organize users & workloads"
...
+
+### Create an environment
+
+Any user with the **User** role or the *Environments: Create* permission can add additional environments and decide who should have access to it.
+
+1. Click on *Add environment*.  The *Add environment* page will appear.
+1. Enter a name and an optional description for the environment.
+1. If you wish to allow users from other organizations to be added as members, select **Allow external members**.
+1. Click *Submit*.  The environment will be provisioned.
+1. The *Manage members* page appears.  Here you may:
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply en/basic-concepts/environments-to-organize-workloads-and-users.md:29]: I changed the sentence to make it more action-oriented.  I hear what you're saying, I might change this sentence again later.

File: en/basic-concepts/environments-to-organize-workloads-and-users.md [hunk 9]
```diff
@@ -3,12 +3,48 @@ title: "Using Environments to organize users & workloads"
...
+   - select one or more individual users from the list
+   - search for specific users
+   - add **all users** in the organization (auto-membership)
+   - add no users to your environment (click *Skip* to proceed to the next page)
+1. For any users you've added, select an environment role, and if you've enabled auto-membership, select a default environment role.
+1. Click *Apply* to commit the selected members.  The users will be added.
+1. The *Initialize environment* page appears.  Here you may:
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply en/basic-concepts/environments-to-organize-workloads-and-users.md:36]: I agree on this too.  Changing the sentence.

File: en/basic-concepts/environments-to-organize-workloads-and-users.md [hunk 10]
```diff
@@ -3,12 +3,48 @@ title: "Using Environments to organize users & workloads"
...
+   - Configure an isolated network with no access to any other subnet, nor to the public Internet
+   - Configure one, two, or three VPCs.  See [What is a VPC](what-is-a-vpc.md) for more information on VPCs
+   - Configure no network (click *Skip* to finish creating the environment)
+1. If you've chosen to create one or more networks, enter the requested parameters for the networks to create.
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply en/basic-concepts/environments-to-organize-workloads-and-users.md:40]: Agreed!

File: en/basic-concepts/environments-to-organize-workloads-and-users.md (hunk 1 above)
Comment (sweetsweetstefan) [reply en/basic-concepts/environments-to-organize-workloads-and-users.md:48]: Done!

File: fr/administration/rbac.md (hunk 2 above)
Comment (sweetsweetstefan) [reply fr/administration/rbac.md:80]: Yeah!

**Pull request merged**